src/foam/nanos/auth/UserAndGroupAuthService.js

```diff
@@ -298,6 +309,11 @@ foam.CLASS({
           throw new AuthenticationException("User not found");
         }
 
+        // check that the user is active
```

Comment:
Since repeated 3 times, could be moved to a method like assertUserIsActive() or something.